### PR TITLE
Add CI workflow

### PR DESCRIPTION
**Pull request by majordutch**
- Fix test engine path
- Fix TestContract ctor to work with latest neo3-boa
- Add requirements.txt
- Update README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,35 +25,39 @@ This contract features two methods to handle properties:
...
-test_ghost.py
+python -m unittest test_ghost
 ```
 
-individual test can be run witn  
+Individual test can be run with:
+
 ```
-python -m unittest test_ghost.GhostTest.test_ghost_decimals
+python -m unittest test_ghost.GhostTest.test_gm_decimals
```
Comment (vincentgeneste) [inline README.md]: should be `test_ghost.GhostTest.test_ghost_decimals`

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      uses: actions/setup-dotnet@v2
+      with:
+        dotnet-version: 6.0.x
+
+    - name: Check out neo devpack
+      uses: actions/checkout@v3
+      with:
+        repository: simplitech/neo-devpack-dotnet
+        path: neo-devpack-dotnet
+        ref: v3.2.1
```
Comment (vincentgeneste) [inline .github/workflows/ci.yml]: is it the latest?

File: tests/test_ghost.py [hunk 3]
```diff
@@ -45,7 +45,7 @@ def build_contract(self, preprocess=False):
             self.CONTRACT = hash160(output)
 
     def deploy_contract(self, engine):
-        cc = TestContract(self.CONTRACT_PATH_NEF)
+        cc = TestContract(self.CONTRACT_PATH_NEF, self.CONTRACT_PATH_JSON)
```
Comment (vincentgeneste) [inline tests/test_ghost.py:48]: this was not changed on test_gm.py, any reason?

**Review by vincentgeneste — COMMENTED** [review]
few minor tweaks to check

File: tests/test_ghost.py (hunk 3 above)
Comment (majordutch) [reply tests/test_ghost.py:48]: I do not see this method signature in test_gm.py

Comment (vincentgeneste) [reply tests/test_ghost.py:48]: in your last push, on `test_ghost.py` you tweaked

deploy_contract() from `cc = TestContract(self.CONTRACT_PATH_NEF)` to `cc = TestContract(self.CONTRACT_PATH_NEF, self.CONTRACT_PATH_JSON)` so i was wondering why on `test_gm.py` on deploy_contract() we didnt do the same?

**Pull request merged**